association/privacy.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: default
+title: "Membership"
+---
+
+# Privacy Policy
+
+
+**Website**
+ * We do not track users on the website. The website is hosted by GitHub Pages (their privacy statement).
+ * We do not use any cookies on the website.
+
+ * We store the contact data of authors and co-authors. And the contact information of the presenting authors will be made public on GitHub and on this website (this is, however, also stated separately in the Indico submission form).
```

Comment:
For things that are made actually public, would emphasize that it will be *published* unless they ask us not to, and we can remove it from the current version but not from history.  Published things don't have privacy protection.

Reply:
This part was not intended to be there and has been removed.

This is nevertheless a good point. It could be worth making this clarification in the "Author information" section.